src/templates/understandTopic.html

```diff
@@ -49,6 +49,9 @@ <h2 class="bolded-blue" data-i18n="moral-topic-understand-{{topic}}">###</h1>
                 <p class="bolded-blue fs-5" data-i18n="moral-practice-skip"></p>
                 <button type="button" class="btn btn-secondary me-auto" onclick="skipTraing()" data-i18n="moral-skip"></button>
             </div>
+
+            {{#if isRele}}
+            {{else}}
```

Comment:
please remove it if `if` statment is empty

Reply:
ok